pkgs/development/python-modules/servefile/default.nix

```diff
@@ -0,0 +1,33 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, pyopenssl
+, pytestCheckHook
+, requests
+, tox
+}:
+
+buildPythonPackage rec {
+  pname = "servefile";
+  version = "0.5.3";
+
+  src = fetchFromGitHub {
+    owner = "sebageek";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-/ZEMZIH/ImuZ2gh5bwB0FlaWnG/ELxfBGEJ2SuNSEb8=";
+  };
+
+  propagatedBuildInputs = [ pyopenssl ];
+
+  checkInputs = [ pytestCheckHook requests tox ];
+  # Skip a test which fails in the nix build environment.
```

Comment:
Why does it fail?

Reply:
Oh, good point: The test attempts to connect to a port on localhost which fails in the nix build environment with `OSError: [Errno 22] Invalid argument`. I'll make the comment more explicit.

Reply:
Should be addressed in https://github.com/NixOS/nixpkgs/commit/9eff9db94f6b534d811348001bc448871f9926c6.